product_sequence/README.rst

```diff
@@ -0,0 +1,6 @@
+A module that adds sequence to the product.
+====================================================================
+
+This module allows to associate a sequence to the product reference.
+The reference (default code) is unique (SQL constraint) and required.
+Note: This module is incompatible with nan_product_sequence.
```

Comment:
You can put a contributors section with your name here. There's a suggested template for OCA modules here: https://github.com/OCA/maintainers-tools/blob/master/template/module/README.rst